app/bundles/NotificationBundle/Event/NotificationSendEvent.php

```diff
@@ -1,22 +1,20 @@
 <?php
 /**
- * @package     Mautic
  * @copyright   2016 Mautic Contributors. All rights reserved.
  * @author      Mautic
+ *
  * @link        http://mautic.org
+ *
  * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
  */
-
 namespace Mautic\NotificationBundle\Event;
 
 use Mautic\CoreBundle\Event\CommonEvent;
 use Mautic\EmailBundle\Entity\Email;
 use Mautic\LeadBundle\Entity\Lead;
 
 /**
- * Class NotificationSendEvent
- *
- * @package Mautic\NotificationBundle\Event
+ * Class NotificationSendEvent.
```

Comment:
This should probably be cleaned up a bit to remove unused methods, docblocks, etc from copying over from the EmailBundle